### DISCUSSION TITLE
Is it possible to place the Controls underneath the video instead of overlaying?

### DISCUSSION THREAD

**Discussion by mcarneyhspot** [vidstack/player#1108] [Q&A] [answered]
First off, this library looks awesome and very well structured and I am evaluating as a potential replacement to videoJS for some parts of the application I work on.  However, one issue I am looking at is how if it is possible to place the Controls section (Play, Volume, Sliders etc) underneath the video itself as a separate container as it seemed like all of the examples have it overlaying the video (as well as the default layout).  Is this something that is supported by the library or potentially will be in the future?  Thanks!

**Comment by mihar-22** [comment]
Awesome, glad you're interested in trying out the library! You can definitely have external contols that are placed outside of the video canvas. The provider component (which you can see in the examples inside the player), let's you decide where the video is placed. Simply set `position: relative` on the controls component and go from there. If you get stuck let me know and I can share a simple example :)

**Reply by mcarneyhspot** [reply]
Great, thanks!  If you can provide or have a simple example that would be awesome.

**Comment by wplit** [comment, marked as answer]
Here's a quick example mate.

(see the extra CSS added at the bottom of video-player.css) click on 'video player' to see the result.

https://stackblitz.com/edit/vidstack-examples-xsc8dp?file=src%2Fvideo-player.css

**Reply by mcarneyhspot** [reply]
Exactly what I was looking for, thank you so much for your quick response!
